modules/phase_field/test/tests/spherical_gaussian_bicrystal/tests

```diff
@@ -0,0 +1,20 @@
+[Tests]
+  [Epsilon_model_out]
+    method = '!DBG'
+    type = 'CSVDiff'
+    input = 'epsilon_model_bicrystal.i'
+    csvdiff = 'epsilon_model_bicrystal_out.csv'
+    requirement = "The system shall be able to compute the accurate anisotropic grain boundary energy and mobility for the epsilon model"
+    design = 'SphericalGaussianKernelAction.md EpsilonModelEpsilonGradientKernel.md EpsilonModelMGradientKernel.md SphericalGaussianMaterial.md'
+    issues = '#27774'
+  []
+  [Gamma_model_out]
+    method = '!DBG'
```

Comment:
```suggestion
```

Reply:
that test runs in 9s we can afford dbg

Reply:
It actually just crash/fail with dbg. I had it for all my tests previously and it would cause an error too. Not sure why, I will look into it later. So, I ended up doing this so that the checks can pass.

Reply:
This likely indicates a bug. Probably an OOB access

You ll want to compile in dbg (set METHOD to dbg before compiling) then run in a debugger

Reply:
I have not tried using dbg with my latest submission yet. I made lots of changes since that time. 
Let's see if it works with the latest one first.  If not, I will try to fix it on my end.

Please activate.

Reply:
@GiudGiud 
They both run and converge but they just seem to timeout:  
Final Test Results:

"
--------------------------------------------------------------------------------------------------------------
[300.0s]  TIMEOUT phase_field/test:spherical_gaussian_bicrystal.Epsilon_model_out FAILED (TIMEOUT) [previous results: TIMEOUT,FINISHED]
[300.0s]  TIMEOUT phase_field/test:spherical_gaussian_bicrystal.Gamma_model_out FAILED (TIMEOUT) [previous results: TIMEOUT,FINISHED]
--------------------------------------------------------------------------------------------------------------
Ran 2 tests in 798.8 seconds. Average test time 300.0 seconds, maximum test time 300.0 seconds.
0 passed, 0 skipped, 0 pending, 2 FAILED
20 longest running jobs:
--------------------------------------------------------------------------------------------------------------
[300.0s]  TIMEOUT phase_field/test:spherical_gaussian_bicrystal.Gamma_model_out FAILED (TIMEOUT) [previous results: TIMEOUT,FINISHED]
[300.0s]  TIMEOUT phase_field/test:spherical_gaussian_bicrystal.Epsilon_model_out FAILED (TIMEOUT) [previous results: TIMEOUT,FINISHED]
--------------------------------------------------------------------------------------------------------------
"
Should I just reduce the time?

Reply:
Increase the max time

Set
max_time = 500

Once we know how long they take we might make them heavy

Reply:
I added max_time and reduce the running time. 
Please activate.